serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc

```diff
@@ -21,7 +21,7 @@ include::modules/serverless-functions-secrets-configmaps-interactively-specializ
 [id="serverless-functions-secrets-configmaps-manually_{context}"]
 == Adding function access to secrets and config maps manually
 
-You can manually add configuration for accessing secrets and config maps to your function.
+You can manually add configuration for accessing secrets and config maps to your function. This might be preferable to the `kn func config` interactive utility and commands, such as when you have an existing configuration snippet.
```

Comment:
```suggestion
You can manually add configuration for accessing secrets and config maps to your function. This might be preferable to using the `kn func config` interactive utility and commands, such as when you have an existing configuration snippet.
```